.github/dockerfiles/Dockerfile.ubuntu-base

```diff
@@ -4,7 +4,7 @@
 ARG BASE=gitpod/workspace-full
 FROM $BASE
 
-ENV INSTALL_LIBS="zlib1g-dev libncurses5-dev libssl-dev unixodbc-dev libsctp-dev lksctp-tools libgmp3-dev libwxbase3.0-dev libwxgtk3.0-gtk3-dev libwxgtk-webview3.0-gtk3-dev"
```

Comment:
Can you add the pkg-config to the list of tools on line 18 instead? The INSTALL_LIBS are meant to be the libraries that are needed at build and run-time.

Reply:
@garazdawi Done.